os/tools/config_util.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+###########################################################################
+#
+# Copyright 2022 Samsung Electronics All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+###########################################################################
+
+import os
+import sys
+
+os_folder = os.path.dirname(__file__) + '/..'
+
+def get_value_from_file(file_name, target):
```

Comment:
Because they will be provided as common APIs, we need to explain what they are. Could you add some comments at each API?

Reply:
Added the description, Thanks.